install/docker/index.markdown

```diff
@@ -73,22 +85,22 @@ You can link a folder on your hard disk to the Docker container, so you can edit
 
 Use this option when you first create the container (or after deleting it with `docker rm my-moveit-container`), like this:
 
-    ./gui-docker -c my-moveit-container -v ~/moveit_ws:/root/linked_moveit_ws
+    ./gui-docker -c my_container -v ~/moveit_ws:/root/linked_moveit_ws
 
-After the container is created it will stay linked, so you can simply enter it on subsequent calls:
+After the container is created, the folder will stay linked. So you can simply enter it on subsequent calls like this:
 
-    ./gui-docker -c my-moveit-container
+    ./gui-docker -c my_container
 
-Files created inside the Docker container are owned by root. Use `chown . -R 1000:1000` inside the container to assign files to the host user if you run into access issues.
+Files created inside the Docker container are usually owned by root. Check and fix permissions, if you run into issues.
```

Comment:
1000 is the default user id of the first user created in Debian-like distros. However, in many situations, this ID might be plain wrong.